mobile/native/src/ln_dlc/node.rs

```diff
@@ -364,6 +364,12 @@ impl Node {
             tokio::time::sleep(reconnect_interval).await;
         }
     }
+
+    pub fn register_usdp_invoice(&self, payment_hash: &bitcoin::hashes::sha256::Hash) {
+        let mut invoices = self.inner.pending_usdp_invoices.lock();
+        invoices.insert(*payment_hash);
```

Comment:
🤔 Should we consider cleaning up that list on received payments?

Reply:
I figured it's in memory only so not hurting, but I guess if it's in the db we should